frontend/components/dashboard/creation/DashboardCreationNetworkMask.vue

```diff
@@ -3,14 +3,16 @@ import { IconNetwork } from '#components'
 import { ChainInfo, ChainIDs } from '~/types/network'
 import { useNetworkStore } from '~/stores/useNetworkStore'
 
+const { t: $t } = useI18n()
+
 const { currentNetwork, isMainNet } = useNetworkStore()
 
 // TODO: get the list from the API...
 let ValidatorDashboardNetworkList: ChainIDs[]
 if (isMainNet()) {
   ValidatorDashboardNetworkList = [ChainIDs.Ethereum, ChainIDs.Gnosis]
 } else {
-  ValidatorDashboardNetworkList = [ChainIDs.Holesky, ChainIDs.Chiado]
+  ValidatorDashboardNetworkList = [ChainIDs.Chiado, ChainIDs.Holesky]
```

Comment:
**Question:** What is the reason for this change?

Reply:
as noted in the ticket: active Buttons should appear left